tests/providers/test_address.py

```diff
@@ -166,6 +166,11 @@ def test_street_suffix(self):
         assert isinstance(suffix, string_types)
         assert suffix in FiProvider.street_suffixes
 
+    def test_state(self):
+        state = self.factory.state()
+        assert isinstace(state, string_types)
```

Comment:
typo: `isinstance` (last `n` is missing)

Reply:
Thanks for notifying me. That's been now fixed. :+1: